lib/workers/repository/update/pr/index.ts

```diff
@@ -57,6 +64,37 @@ export type ResultWithoutPr = {
 
 export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 
+export function updatePrRenovateVerData(
```

Comment:
when do i do the creation then?

Reply:
if i understand correctly, u want to do the rendering of `updatedByRenovateVersion` always when we call getBody regardless! 
But for the creation it should be added only for creating new PRs?

Reply:
Well, I think for simplicity we can just set `createdByRenovateVersion` to be equal `updatedByRenovateVersion` when nothing has been provided. It will be a way simpler than tracking *unknown* creation date, and all the `debugData` fields can remain mandatory.

Reply:
In other words, every unknown `createdByRenovateVersion` will be equal the version of this PR release, but week later this won't matter.

Reply:
so i shouldn't render data anywhere else! just `getPrBody`, if `createdByRenovateVersion`  exists, i keep it, if not i fill it with current version!, 

let me do that sec
i think i broke some /Branch/ tests for some reason xD